packages/foam-vscode/src/features/preview-navigation.ts

```diff
@@ -29,13 +33,24 @@ export const markdownItWithFoamLinks = (
     regex: /\[\[([^[\]]+?)\]\]/,
     replace: (wikilink: string) => {
       try {
-        const resource = workspace.find(wikilink);
-        if (resource == null) {
-          return getPlaceholderLink(wikilink);
+        let linkHasAlias = wikilink.includes('|');
+        let resourceLink = wikilink;
+        if (linkHasAlias) {
+          resourceLink = wikilink.substring(0, wikilink.indexOf('|'));
+        }
```

Comment:
minor: 

```suggestion
        const resourceLink = linkHasAlias  
          ? wikilink.substring(0, wikilink.indexOf('|'))
          : wikilink;
```

for two reasons:
1. let's use `const` whenever possible
2. this is easier to read